### ISSUE TITLE
Icons should be affected by gravity

### ISSUE THREAD

**Issue by osrl** [rengwuxian/MaterialEditText#131]
When the edit text's gravity is set to top, iconLeft is still on the bottom. Maybe there is something i've missed

**Comment by rengwuxian** [comment]
Hi @osrl ,

Different developers may want different alignments, even if the gravity is set to `top`. To set the icon aligned with the bottom line is by design.

**Comment by osrl** [comment]
By design do you mean material design? Can you please show me a sample on design pattern? I thought they are center aligned in [this](http://material-design.storage.googleapis.com/publish/v_2/material_ext_publish/0Bx4BSt6jniD7NF9BVjcxdUlJR0U/components_textfields_labels2.png) example. Am I wrong?

**Comment by rengwuxian** [comment]
Haha It's only by my design. I just can't set out a perfect rule about where to put the icon. For example, I don't know whether I should put the icon aligned at the floating label's top, or the main text's top, or the EditText's edge when the gravity is set to `top`.

**Comment by osrl** [comment]
Yeah, you are right about `top`. I think at least it should be center aligned by default.

**Comment by rengwuxian** [comment]
Could you please do this work? I even don't get a chance to figure out a rule to show the "center" gravity.

**Comment by veelck** [comment]
Maybe it would be possible to add appropriate config params to let user/developer decide where he'd like the icon to appear?
